### DISCUSSION TITLE
AMD R7 7840HS Device

### DISCUSSION THREAD

**Discussion by IcedSoyaBeanMilk** [ggml-org/llama.cpp#7004] [Q&A]
I possess a device equipped with an AMD R7 7840HS CPU and no discrete graphics card. Which build from the Release should I select to achieve the fastest inference speed?

**Comment by Djip007** [comment]
on linux+hipblas I will get source code and rebuild it and use like that:
```sh
# build:
export HSA_OVERRIDE_GFX_VERSION=11.0.1
make -j16 LLAMA_HIPBLAS=1 LLAMA_HIP_UMA=1 AMDGPU_TARGETS=gfx1101
```
```sh
# run:
HSA_OVERRIDE_GFX_VERSION=11.0.1  ./main -ngl 999 --no-mmap [...]
```
